marklogic-data-hub/src/test/java/com/marklogic/hub/hubcentral/AllArtifactsProject.java

```diff
@@ -22,9 +22,9 @@
  */
 public class AllArtifactsProject extends TestObject {
 
-    private HubClient hubClient;
-    private Map<String, JsonNode> zipEntries;
-    private File zipFile;
+    protected HubClient hubClient;
+    protected Map<String, JsonNode> zipArtifactEntries;
```

Comment:
Let's do "projectUpdateZipEntries" to make it clear this is the zip of project updates, not the full project zip.